test/unit/MyAnimeList.test.js

```diff
@@ -0,0 +1,49 @@
+const chai = require('chai');
+const assert = chai.assert;
+const sinon = require('sinon');
+const sinonTestFactory = require('sinon-test');
+const sinonTest = sinonTestFactory(sinon);
+const fs = require('fs');
+const path = require('path');
+const { MyAnimeList, Anime } = require('../../data.js');
+const popura = require('popura');
+
+describe('MyAnimeList', function() {
```

Comment:
In the future, we will probably want a `TestRunner` class so we can share common testing utility functions and share common dependencies